app/controllers/rhapsody_realities.js

```diff
@@ -84,15 +84,22 @@ class RhapsodyController {
         let viewData = {
             menuActive: 'rhapsody'
         };
+        let query = req.query;
+        //Get key to filter by
+        let filterBy = dateFormat(now, 'mm'); //returns the current month as two digit
```

Comment:
you need the whole thing, i.e month and year, otherwise you will be returning for all the say Octobers of all the year in the system

month and year combination is what you want to ensure result is unique